assets/css/heading.css

```diff
@@ -22,11 +25,17 @@
     font-size:16px;
     font-weight:400;
     color:#111;
+    font-family: 'Open Sans', sans-serif;              
+    src:url('../fonts/Open-sans/OpenSans-Regular.ttf');
```

Comment:
Po co src? To już ustawiałeś w font face.
Przejrzyj co napisałeś, pogoogluj i spróbuj zrozumieć co i jak